pyCGM_Single/pyCGM.py

```diff
@@ -2535,9 +2562,9 @@ def JointAngleCalc(frame,vsk):
 
     Joint Axis store like below form
 
-    Basically, the axis form is like [[origin],[axis]]
-    So, there's origin which define the position of axis
-    and there's Unit vector of each axis which is attach to the origin.
+    The axis form is in the form [[origin],[axis]]
+    Origin defines the position of axis and axis is the unit vector of each
+    axis attached to the origin
```

Comment:
... axis is the direction vector of each axis, where the ( axis[0] (x-axis vector) - origin ) = x-axis unit vector.